src/app/components/Services.tsx

```diff
@@ -0,0 +1,30 @@
+import { useTranslations } from 'next-intl';
+
+import ServiceCard from './ServiceCard';
+import servicesDb from '../db/services.json';
+
+const Services = () => {
+  const t = useTranslations('Services');
+  return (
+    <>
+      <h1 className="font-black mx-8 mt-12 text-4xl xl:text-6xl 2xl:mt-32">
+        {t('servicesTitle')}
+      </h1>
+      <div className="flex flex-wrap gap-8 mx-auto mt-16 justify-center w-3/4 2xl:gap-24 2xl:mt-24">
+        {servicesDb?.map((service) => {
+          return (
+            <ServiceCard
+              key={service.title}
```

Comment:
there can be a problem if two services have same title even it's seems not the thing we want to do here. Just in case to be sure you can use the hook useId() available from react 18
[https://react.dev/reference/react/useId](https://react.dev/reference/react/useId)

Reply:
Added! You have totally reason here!